workers/unity/Assets/Gdk/Core/GameObjectRepresentation/EntityGameObjectManager.cs

```diff
@@ -0,0 +1,32 @@
+using Unity.Entities;
+using UnityEngine;
+
+namespace Improbable.Gdk.Core
```

Comment:
do we want this in core? it seems to me that moving it to a separate feature module would be better

Reply:
This shall be in Core as discussed in the design meetings.